client/blocks/comments/post-comment-list.jsx

```diff
@@ -169,6 +240,13 @@ class PostCommentList extends React.Component {
 		);
 	};
 
+	scrollToComment = () => {
+		const comment = window.document.getElementsByName( window.location.hash.substring( 1 ) )[ 0 ];
+		comment.scrollIntoView();
```

Comment:
good question, i have no answer.  mdn didn't have the answer either.
Do you know of a better guaranteed sync alternative?